include/RunAnalysis.h

```diff
@@ -39,7 +39,7 @@
 
 using namespace llvm;
 
-namespace dataflow {
+namespace Dataflow {
```

Comment:
C++'s convention is all lowercase for namespaces, so I don't think we should make this change. See e.g. https://google.github.io/styleguide/cppguide.html#Namespace_Names